src/components/PublishContainer/PublishedScreen/PublishedScreen.tsx

```diff
@@ -0,0 +1,148 @@
+import prettyBytes from "pretty-bytes";
+import React, { FunctionComponent } from "react";
+import { useConfigContext } from "../../../common/context/config";
+import { useFormsContext } from "../../../common/context/forms";
+import { FailedJobErrors, WrappedDocument } from "../../../types";
+import { Container } from "../../Container";
+import { ProgressBar } from "../../ProgressBar";
+import { Button } from "../../UI/Button";
+import { SvgIcon, SvgIconCheckCircle, SvgIconDownload, SvgIconXCircle } from "../../UI/SvgIcon";
+import { Title } from "../../UI/Title";
+import { PublishedTag } from "../PublishedScreen/PublishedTag";
+
+interface PublishScreen {
+  publishedDocuments: WrappedDocument[];
+  failedPublishedDocuments: FailedJobErrors[];
+}
+
+export const PublishedScreen: FunctionComponent<PublishScreen> = ({
+  publishedDocuments,
+  failedPublishedDocuments,
+}) => {
+  const { setConfig } = useConfigContext();
+  const { setForms, setActiveFormIndex } = useFormsContext();
+
+  const createAnotherDoc = (): void => {
+    setForms([]);
+    setActiveFormIndex(undefined);
+  };
+
+  const onDone = (): void => {
+    setForms([]);
+    setActiveFormIndex(undefined);
+    setConfig(undefined);
+  };
+
+  const getFileSize = (jsonString: string): number => {
+    const m = encodeURIComponent(jsonString).match(/%[89ABab]/g);
+    return jsonString.length + (m ? m.length : 0);
+  };
+
+  let failPublishedDocuments = [] as WrappedDocument[];
+
+  if (failedPublishedDocuments && failedPublishedDocuments.length > 0) {
+    failPublishedDocuments = failedPublishedDocuments.reduce((acc, curr) => {
+      const documentsIssuesInJob = curr.documents;
+      return [...acc, ...documentsIssuesInJob];
+    }, [] as WrappedDocument[]);
+  }
+
+  const downloadErrorLog = (): void => {
+    const errorLog = document.createElement("a");
```

Comment:
Realise this is quite a funky way of making a download button. Why not just move the entire section of `const formattedErrorLog...` outside and then in the JSX below just set the `href` & `download`?

Reply:
ok